### PR TITLE
[Debug] fix resource type test on HHVM

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #19079 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by fabpot** [comment]
Do we know why it broke without any change on our side?

**Comment by xabbuh** [comment]
@fabpot The failing test was added in #19079.

**Comment by xabbuh** [comment]
see https://github.com/symfony/symfony/pull/19079/files#diff-012ea9fdf4f90d7e4582c87039a4fed4R193

**Comment by xabbuh** [comment]
and this simply seems to be the returned value in all HVM versions: https://3v4l.org/D8sLi

**Comment by fabpot** [comment]
Looks like the tests still do not pass on HHVM.

**Comment by xabbuh** [comment]
Tests finally pass (we had to swap the expected and actual arguments for the assertions too).

File: src/Symfony/Component/Debug/Tests/Exception/FlattenExceptionTest.php [hunk 1]
```diff
@@ -193,6 +193,7 @@ public function flattenDataProvider()
     public function testArguments()
     {
         $dh = opendir(__DIR__);
+        $fh = fopen($filename = tempnam(sys_get_temp_dir(), ''), 'r');
```
Comment (rybakit) [inline src/Symfony/Component/Debug/Tests/Exception/FlattenExceptionTest.php]: `$fh = tmpfile();`?

Comment (xabbuh) [inline src/Symfony/Component/Debug/Tests/Exception/FlattenExceptionTest.php]: makes sense

**Comment by xabbuh** [comment]
Tests do now pass again on HHVM.

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**